runestone/parsons/js/parsons.js

```diff
@@ -126,7 +126,7 @@ LineBasedGrader.prototype.grade = function() {
 			       feedbackArea.html("Perfect!  It took you only one try to solve this.  Great job!");
 			    }
 				correct = true;
-				problem.helpButton.disabled = false;  // bje
+				// problem.helpButton.disabled = false;  // bje
```

Comment:
@ericsonga, Hillary is correct when I try to check an answer that is correct, this error stops the chain, and then logging does not happen for correct answers!  I wonder if you really meant to say this.helpButton.disabled = false; ???